### DISCUSSION TITLE
Welcome to Vidar Discussions!

### DISCUSSION THREAD

**Discussion by clabe45** [etro-js/etro#41] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share how you use Vidar.
  * Give constructive feedback.
  * Engage with other community members.
  * Welcome others and are respectful to everyone! Remember that this is a community we
  build together 💪

  To get started, comment below with an introduction of yourself and tell us about why you are interested in Vidar!

**Comment by RahulLanjewar93** [comment]
I am getting the error 
"Play was interrupted by pause" , when I use movie.play()

Though the video plays in the video element , it is not visible in canvas element which was passed to movie function . Any fixes?

**Reply by clabe45** [reply]
Are you calling movie.play() after movie.pause()?

**Reply by RahulLanjewar93** [reply]
I never called pause I am just calling play

**Reply by RahulLanjewar93** [reply]
Also how can I stretch the video's width and height so that it takes up the remaining space of the canvas
tried editing **_height,sourceHeight,destHeight_** but none of them works

**Reply by clabe45** [reply]
Ah, please open an issue for the "Play was interrupted by pause" error so contributors can see it.

What do you set those height properties to? Setting either height or destHeight to the movie's height should work. Height defaults to destHeight, which defaults to sourceHeight. Please let me know if that doesn't work.
